src/bot/modules/stickys.ts

```diff
@@ -36,11 +36,15 @@ export default class Stickys {
 
     const message = messages.find((m) => m.author.id === this.client.user!.id && m.embeds?.[0]?.title === embed.title);
     const last_message = messages.first();
+
     if (!message || !last_message || message.id !== last_message.id) {
       if (message) await message.delete();
 
       await channel.send({ embeds: [embed] });
     }
+    if (message?.id === last_message?.id) {
+      await message?.edit({ embeds: [embed] });
+    }
```

Comment:
Je pense que c'est plus propre.
```suggestion
    if (message && last_message && message.id === last_message.id) {
      await message.edit({ embeds: [embed] });
    } else {
      if (message) await message.delete();
      await channel.send({ embeds: [embed] });
    }
```